process/css-styleguide/README.md

```diff
@@ -0,0 +1,168 @@
+
+## About the style guides
+
+The style guide consists of a number of common UI components such as buttons, headings, search inputs that can be included while creating the UI of a project. It is based on the [Meilisearch Design System](https://www.figma.com/file/XXmUvsZzfZBKt1i2v12wf4/Design-System?node-id=1504%3A282).
+
+For creating beautiful in-brand demos, we encourage you to consult the [Meilisearch Guide to Design a Demo](https://www.figma.com/file/YUqMlOmApYd0D2aitPyDce/Guide-to-design-a-demo?node-id=2%3A8).
+
+## Components included
+
+- Button
+- Input
+- Logos
+- Modal
+- Search Input
+- Tag
+- Typography: headings and body text
+
+## Usage
+
+### Button
+
+1. The base class for styling a button is `.btn`.
+2. Change the color styling of the button by using the following classes:
+- `.btn-dodger-blue` 
+- `.btn-hot-pink`
+- `.btn-lila` 
+- `.btn-turquoise`
+- `.btn-watermelon`
+- `.btn-peppermint`
+- `.btn-candlelight`
+
+```html
+<button class="btn btn-dodger-blue" type="button">Button</button>
+```
+
+3. You can create a secondary button by adding the `btn-secondary` class
+
+```html
+<button class="btn btn-dodger-blue btn-secondary" type="button">Button</button>
+```
+
+4. To change the size of the button, you can add `btn-sm` or `btn-lg` to the class list.
+
+```html
+<button class="btn btn-sm btn-dodger-blue" type="button">Button</button>
+```
+
+
+Output :
+
+<img width="50%" alt="Capture d’écran 2022-08-23 à 10 33 08" src="https://user-images.githubusercontent.com/48251481/186461703-49f5d19b-128e-4b28-9348-f1838c69ab74.png">
+
+
+### Input
+
+1.  The parent class `.input` will assign some basic styling to the input.
+2.  Labels can also be styled depending upon various needs. The classes are `.input--label`, `.input--message`, `.input--error`, and `.input--message-error`
+
+```html
+<input
+  type="text"
+  class="input input--label"
+  placeholder="Write something"
+/>
+```
+
+Output :
+
+<img width="311" alt="Capture d’écran 2022-08-23 à 10 34 05" src="https://user-images.githubusercontent.com/48251481/186462336-24f8c55e-29fc-49c8-93c4-e45a90eb1e72.png">
+
+### Modal
+
+1. To render the content in a modal window, wrap it inside the parent element such as `div` and give it the `.modal` class.
+
+```html
+<div class="modal">
+    <div class="title title-l">This is a modal</div>
+</div>
+```
+
+Output :
+
+![Screenshot from 2022-07-11 15-08-37](https://user-images.githubusercontent.com/64376712/178236255-d999514e-50f9-4960-9958-497ce207d1ed.png)
+
+### Search form
+
+The search form is composed of 3 elements, each with its own corresponding classes:
+1. A form with a `.search-form` class, which gives flex container properties to the form element
+
+2. An input: the base class `.input` assign basic input style to the input. It needs to be combined with `search-input` to give it a search bar style, adding the magnifier glass to it.
+
+3. A reset button: the base class for styling a reset button is `.search-input-reset`.
```

Comment:
Using Chrome, I have 2 reset button displaying (I just opened the `css-styleguide/search-form/index.html` file):

<img width="138" alt="Capture d’écran 2022-08-29 à 11 40 02" src="https://user-images.githubusercontent.com/30866152/187172865-5ee05f8a-6de1-4263-ba64-643d934f8693.png">

Reply:
🤔 that's weird. I can't reproduce the issue in my chrome browser...

Reply:
Oh well, that's weird indeed 🙈

Here is what I get on Chrome: 

https://user-images.githubusercontent.com/30866152/187388383-d4c00dd0-9b15-4d62-b7a9-406e257d86dc.mov

I also have a few other problems with this component:

- When I type something and press the esc. button on my keyboard, the input is not cleared. This is bad for accessibility 🙈 The default behavior of the `<input type="search">` does this, but here, since we "trick" the clear button with a custom one inside a container, it doesn't work anymore. I'm not sure how you would like to use this `search-form` style, but if possible I would totally recommend using the default `clear` button and updating it with the styles you want :) 
- The outline is quite ugly, can we do something about it?
<img width="96" alt="Capture d’écran 2022-08-30 à 10 27 37" src="https://user-images.githubusercontent.com/30866152/187388623-959f7d16-6931-406a-922b-485cb77b2ab8.png">